#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-extra-jars-in-new-location.dpatch by Tiago Saboga <tiagosaboga@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not specify paths in Class-path.

@DPATCH@
diff -urNad omegat-1.8.1.2+dfsg~/manifest-template.mf omegat-1.8.1.2+dfsg/manifest-template.mf
--- omegat-1.8.1.2+dfsg~/manifest-template.mf	2009-02-09 21:03:32.000000000 -0200
+++ omegat-1.8.1.2+dfsg/manifest-template.mf	2009-02-09 21:09:50.000000000 -0200
@@ -3,7 +3,7 @@
 License: GNU Public License version 2
 Specification-Version: 1
 Implementation-Version: @version@
-Class-Path: lib/lib-mnemonics.jar lib/htmlparser.jar lib/MRJAdapter.jar lib/vldocking_2.1.4.jar lib/jna.jar lib/swing-layout-1.0.jar lib/retroweaver-rt-2.0.1.jar lib/jmyspell-core-1.0.0-beta-2-1.4.jar lib/backport-util-concurrent.jar
+Class-Path: omegat-lib-mnemonics.jar libhtmlparser.jar vldocking.jar jna.jar swing-layout.jar
 
 
 
